### PR TITLE
docs(general): SEO optimizations for WPFtoAngular #1447

### PR DESCRIPTION
**Pull request by rmkrmk**
#1447

### CODE REVIEW

File: en/components/general/wpf_to_angular_guide/angular_events.md [hunk 1]
```diff
@@ -1,16 +1,14 @@
 ---
-title: Angular Events
-_description: Learn how to respond to events in Angular and implement your own events.
-_keywords: WPF, Windows Presentation Foundation, Angular, Ignite UI for Angular, Events, Event Binding
+title: Handling Events in Angular | IgniteUI for Angular | Infragistics
```
Comment (zdrawku) [inline en/components/general/wpf_to_angular_guide/angular_events.md]: `angular events` would be a better choice for a primary keyword

File: en/components/general/wpf_to_angular_guide/angular_events.md [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
+_description: Learn how to handle events in Angular when migrating from WPF to Angular.
```
Comment (zdrawku) [inline en/components/general/wpf_to_angular_guide/angular_events.md]: The description should be up to 160 characters and should use the defined primary and secondary keywords

File: en/components/general/wpf_to_angular_guide/angular_events.md [hunk 3]
```diff
@@ -1,16 +1,14 @@
...
 
-## Responding to Events
+## Handling Events in Angular
 
 > [!Video https://www.youtube.com/embed/V1Futz4W400]
 
-### Binding to User Input Events
-
-Every app you have probably ever written had to respond to some type of user interaction one way or another. The most common way to do that is with some type of event system. WPF provides routed events, CLR events, and commands. While in Angular, there are DOM events.
+The Angular events are emitted as a response to a user interaction. When an Angular event is emitted this triggers the logic contained in the event handler. WPF provides routed events, CLR events, and commands. While in Angular, there are DOM events.
```
Comment (zdrawku) [inline en/components/general/wpf_to_angular_guide/angular_events.md]: All keywords should be used in the first paragraph of the topic

File: en/components/general/wpf_to_angular_guide/angular_events.md (hunk 1 above)
Comment (rmkrmk) [reply en/components/general/wpf_to_angular_guide/angular_events.md]: Addressed.

File: en/components/general/wpf_to_angular_guide/angular_events.md (hunk 2 above)
Comment (rmkrmk) [reply en/components/general/wpf_to_angular_guide/angular_events.md]: Addressed.

File: en/components/general/wpf_to_angular_guide/angular_events.md (hunk 3 above)
Comment (rmkrmk) [reply en/components/general/wpf_to_angular_guide/angular_events.md]: Addressed.

File: en/components/general/wpf_to_angular_guide/angular_events.md [hunk 4]
```diff
@@ -1,16 +1,14 @@
 ---
-title: Angular Events
-_description: Learn how to respond to events in Angular and implement your own events.
-_keywords: WPF, Windows Presentation Foundation, Angular, Ignite UI for Angular, Events, Event Binding
+title: Handling Angular Events | IgniteUI for Angular | Infragistics
+_description: Learn how Angular events handing works when you migrate from WPF to Angular.
```
Comment (zdrawku) [inline en/components/general/wpf_to_angular_guide/angular_events.md]:
```suggestion
_description: Learn how Angular events handling works when you migrate from WPF to Angular.
```

**Pull request merged**